modules/announcements/assets/js/components/AnnouncementBody.js

```diff
@@ -0,0 +1,28 @@
+import PropTypes from 'prop-types';
+
+export default function AnnouncementBody( { announcement } ) {
+	const { title, description, media } = announcement;
+
+	return (
+		<div className="announcement-body-container">
+			<div className={ `announcement-body-media announcement-body-${ media.type }` }>
+				{ 'image' === media.type && (
+					<img src={ media.src } alt="Announcement" />
+				) }
+			</div>
+			<div className="announcement-body-title">
+				{ title }
+			</div>
+			<div className="announcement-body-description">
+				{ description }
+			</div>
+		</div>
+	);
+}
+
+AnnouncementBody.propTypes = {
+	announcement: PropTypes.object.isRequired,
+	title: PropTypes.string,
+	description: PropTypes.string,
+	media: PropTypes.object,
```

Comment:
maybe I am missing something, but these are not properties, they are passed inside one, if you want them to be properties maybe do the destructuring during the Component call rather than at line 4

Reply:
They are passed inside announcement and need to be declared